src/table/sticky-columns/__tests__/use-sticky-columns.test.tsx

```diff
@@ -321,3 +322,19 @@ describe('getStickyClassNames helper', () => {
     });
   });
 });
+
+test('updateCellOffsets element widths fallback to 0 when elements are missing', () => {
```

Comment:
updated

Reply:
The ask was about to find a way to test it high-level, by interacting with the table as a whole component as opposed to  calling this function `updateCellOffsets` directly.

I approved because we probably should get the fix released as soon as possible. 

But I would appreciate a follow up PR with a proper test

Reply:
The current revision is not approved. Please do so.

Do you mind if I implement the integration test for this? Testing sticky columns with a table in unit tests is tricky as it requires careful mocks of the getBoundingClientRect.